docs/UserGuide.md

```diff
@@ -21,251 +45,323 @@
+       
+15. You've finished a task. Congrations! Let's mark a task as finshed.   
+    * `i` to view index
+    * `fin 3`
+    Dateline task learn task-tracker is marked as finished. 
+16. To exit the program, try:
 
 	* `exit`
 
       Close the Task-tracker.
 
-14. Refer to the [Features](#features) section below for details of each
+17. Refer to the [Features](#features) section below for details of each
     command.
+    
+<a name="qss">**Quick Start Summary**</a>
+
+| Command | Essential Parameters|
+|---------|:----------|
+|add| `"taskname"` <br> `"deadline task name"` `date` `time` <br> `"event name"` `start date` `start time` `end date` `end time`|
+|i|  |
+| edit | `floating task index` `n-` `p-` <br> `deadline task index` `n-` `dd-` `dt-` <br> `event index` `n-` `sd-` `st-` `ed-` `et` `loc-` <br> 
+| del | `index`|
+| fin |`index`| 
+| view | `date` <br> `events` |
+| exit |          |
+    
+ 
 
 ## Features
 
-**Command Format**
+<a name="dm">**Data models**</a>
+The task stored in Task Tracker will be automatically grouped into three different type of task: floating task, deadline task, and event, depending on the type and number of fields entered when creating the task. 
+
+|A/An... | has...|
+|----| :--------:|:------|
+|Floating Task | only a task name|
+|Deadline task |  end time and date |
+|Event | start time and date,  end time and date|
 
-* Words in `UPPER_CASE` are the required parameters.
+All the time and date fields of the task have to be entered in order to create the task.
+
+<a name="cf">**Command Format**</a>
+
+* Words in `UPPER_CASE` are parameters to be defined by the user.
 
 * Words in `lower_case` are the reserved keywords.
 
 * Items in `[SQUARE_BRACKETS]` are optional parameters.
 
-* Items in `<ANGLE_BRACKETS>` are partially optional parameters (a minimum
-  number of partial optional parameters are required for certain command).
+* Items in `<ANGLE_BRACKETS>` essential parameters. All essential parameters have to be included in the command.
 
 * Items separated by `|` are in parallel relation, only one of them should be
   use in each command.
 
 * Items with `...` after them can have multiple instances.
 
-### Viewing help : `help`
+* The parameters have to follow the order in which they are presented, unless otherwise specified.
 
-    help
+<a name="pk">**parameter keywords**</a>
 
-Help is also shown if you enter an incorrect command e.g. `abcd`
+|Keyword | Definition|
+|----| :--------:|:------|
+|`tdy` | today|
+|`tmr` | tommorow|
+|`yst` | yesteday|
+| `to` | to |
+|`jan` | January|
+|`feb` | February|
+|`mar` | March|
+|`apr` | April|
+|`may` | May|
+|`jun` | June|
+|`jul` | July|
+|`aug` | August|
+|`sep`| September|
+|`oct` | October|
+|`nov` | November|
+|`dec` | December|
 
-### Adding a floating task : `task`
+<a name="DateFormat">**Date Format**</a>
+Valid input examples:
 
-Adds a floating task to TaskTracker.
+ddmonthyy
 
-    task "FLOATING_TASK_NAME" [PRIORITY]
+	31dec2016
```

Comment:
I preferred the numbers and slashes. e.g. `31/12/2016`. Can we revert back to that for now?